docs/admin/accessing-the-api.md

```diff
@@ -52,8 +52,10 @@ On GCE, Client Certificates, Password, Plain Tokens, and JWT Tokens are all enab
 If the request cannot be authenticated, it is rejected with HTTP status code 401.
 Otherwise, the user is authenticated as a specific `username`, and the user name
 is available to subsequent steps to use in their decisions.  Some authenticators
-may also provide the group memberships of the user, while other authenticators
-do not (and expect the authorizer to determine these).
+also provide the group memberships of the user, while other authenticators
+do not.  An authorizer may determine as well as use groups (or any other
```

Comment:
@lavalamp Thanks for the review.  I am not sure I understand it.  Why do you think the reader might imagine group information comes from something besides the authenticator or internal activity of the authorizer?  What exactly do you mean by "source"?  Clearly some code running in the apiserver can not itself be an "ultimate" source, it could only derive group information by some means from some thing(s).  I am not sure I understand the distinction you are asking about between "use groups from some other source" vs "itself be a source of groups".

I fear we may be working too hard to document something that is, as far as I know, entirely hypothetical (no authorizers known to me behave as we are trying to document).  Perhaps the better part of valor is to simply delete the hypothetical discussion, and let the text end with "Some authenticators also provide the group memberships of the user, while other authenticators do not".